trendly/src/app/clusterly/clusterly.component.css

```
@@ -0,0 +1,20 @@
+.clusters-container {
+  margin: auto;
+  padding: 10px;
+}
+
+::ng-deep .bubble-tooltip {
+  display: none;
+  background-color: black;
+  color: white;
```

Use hex values for CSS colors (or rgb/rgba when needed)
http://web.simmons.edu/~grovesd/comm244/notes/week3/css-colors#rgb
```suggestion
  background-color: #000;
  color: #fff;
```

---

Done.